login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Login</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="login-container">
+      <form class="login-form">
+        <img class= "logo" src="img/logo.png">
+        <div class="input-elements">
+          <label class="label">Email</label>
+          <input type="text" class="input">
+        </div>
+        <div class="input-elements">
+          <label class="label">Password</label>
+          <input type="password" class="input">
+        </div>
+        <div class="form-actions">
+          <a href="list.html" class="fill-btn">Login</a>
+          <button class="no-fill-btn">Sign up</button>
```

Comment:
I thought about it but I think that the `type="submit"` button should be the Login button and not the Sign up one (since it's the login form). The issue is that the cards asks me to use the Login button to do a redirection instead of submitting the form

Reply:
ohh yes, i see. ok then add type=button. It is good practice to add a type to all buttons for seo and accesibility reasons